src/Compilers/CSharp/Portable/Syntax/SyntaxFacts.cs

```diff
@@ -381,5 +381,65 @@ internal static bool IsVar(this Syntax.InternalSyntax.SyntaxToken node)
         {
             return node.Kind == SyntaxKind.IdentifierToken && node.ValueText == "var";
         }
+
+        /// <summary>
+        /// Figures out if this expression is a type in a deconstruction-declaration.
+        /// Outputs the VariableDeclarationSyntax directly containing it, if that is the case.
+        /// </summary>
+        internal static bool IsDeconstructionType(ExpressionSyntax expression, out SyntaxNode parent)
```

Comment:
I had that and changed it back. It was making the call site more trouble. I'll take another look.